pgmq-extension/README.md

```diff
@@ -41,24 +41,25 @@ Postgres 12-16.
     - [Archive a message](#archive-a-message)
     - [Delete a message](#delete-a-message)
     - [Drop a queue](#drop-a-queue)
-- [Configuration](#configuration)
-  - [Partitioned Queues](#partitioned-queues)
+  - [Configuration](#configuration)
+    - [Partitioned Queues](#partitioned-queues)
   - [Visibility Timeout (vt)](#visibility-timeout-vt)
+  - [Who uses pgmq?](#who-uses-pgmq)
   - [✨ Contributors](#-contributors)
 
 ## Installation
 
 The fastest way to get started is by running the Tembo Docker image, where PGMQ comes pre-installed in Postgres.
 
 ```bash
-docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pg16-pgmq:latest
+docker run -d --name pgmq-postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pg16-pgmq:latest
 ```
 
 If you'd like to build from source, you can follow the instructions in [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ### Updating
 
-To update PGMQ versions, follow the instructions in [UPDATING.md](UPDATING.md).
+To update PGMQ versions, follow the instructions in [UPDATING.md](pgmq-extension/UPDATING.md).
```

Comment:
Does UPDATING.md need to be moved into the `pgmq-extension` directory?

Reply:
It moved in https://github.com/tembo-io/pgmq/pull/287